### PR TITLE
[MAT-6576] Use Spotify Formatter

### PR DESCRIPTION
**Pull request by jkotanchik-SB**
## MADiE PR

Jira Ticket: [MAT-6576](https://jira.cms.gov/browse/MAT-6576)
(Optional) Related Tickets:

### Summary

Use Spotify formatter since it supports newer Java feature syntax.

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included and Code coverage is verified.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 spring:
   data:
     mongodb:
-      database: ${MONGO_DATABASE:madiecqllibrary}
-      uri: ${MONGO_URI:mongodb://${DBUSER}:${DBPASS}@localhost:27017/madiecqllibrary}?authSource=admin&maxPoolSize=50&connectTimeoutMS=2000&serverSelectionTimeoutMS=2000
+      database: ${MONGO_DATABASE:madie}
+      uri: mongodb://root:E5press0@${MONGO_HOST:localhost}:27017/madie?authSource=admin&maxPoolSize=50&connectTimeoutMS=2000&serverSelectionTimeoutMS=2000
```
Comment (nmorasb) [inline src/main/resources/application.yml]: was this intentionally added? Should we maybe make `root` a default value for the `${DBUSER}` var? (And same for `DBPASS`)

Comment (jkotanchik-SB) [reply src/main/resources/application.yml]: Nope! Accidental add of local config, I'll clear it out.

**Review by nmorasb — APPROVED** [review]
